src/main/java/com/amadeus/resources/DatedFlight.java

```diff
@@ -0,0 +1,94 @@
+package com.amadeus.resources;
+
+import lombok.Getter;
+import lombok.ToString;
+
+/**
+ * An DatedFlight object as returned by the On Flight Status Demand API.
+ * @see com.amadeus.schedule.Flights#get()
+ */
+@ToString
+public class DatedFlight extends Resource {
+  protected DatedFlight() {}
+
+  private @Getter String type;
+  private @Getter String scheduledDepartureDate;
+  private @Getter FlightDesignator flightDesignator;
+  private @Getter FlightPoints[] flightPoints;
+  private @Getter Segments[] segments;
+  private @Getter Legs[] legs;
+
+  @ToString
+  public class FlightDesignator {
+    protected FlightDesignator() {
+    }
+
+    private @Getter String carrierCode;
+    private @Getter int flightNumber;
+  }
+
+  @ToString
+  public class FlightPoints {
+    protected FlightPoints() {
+    }
+
+    private @Getter String iataCode;
+    private @Getter Departure departure;
+    private @Getter Arrival arrival;
+  }
+
+  @ToString
+  public class Departure {
+    protected Departure() {
+    }
+
+    private @Getter Timings[] timings;
```

Comment:
Missing gate and terminal: 
<img width="360" alt="image" src="https://user-images.githubusercontent.com/713173/92693267-a8e38f00-f345-11ea-8a84-e9685d94d15a.png">